res/css/views/rooms/_BasicMessageComposer.scss

```diff
@@ -66,6 +65,14 @@ limitations under the License.
                     font-size: $font-10-4px;
                 }
             }
+
+            span.mx_UserPill {
+                cursor: pointer;
+            }
+
+            span.mx_RoomPill {
+                cursor: default;
+            }
```

Comment:
is this necessary? as in, is it not implied by lack of it

Reply:
We get the text cursor thingy without it